database/sqlite_adapter.go

```diff
@@ -50,3 +69,164 @@ func (a *SQLiteAdapter) DoMigrations(db *sqlx.DB, cfg *config.DbConfig) error {
 
 	return nil
 }
+
+func (a *sqLiteAdapter) getDBx() *sqlx.DB {
+	if a.db == nil {
+		panic("connection to the database has not been established")
+	}
+	return a.db
+}
+
+func (a *sqLiteAdapter) importHeaders(inputFile *os.File, log *zerolog.Logger) (int, error) {
+	// prepare db to bulk insterts
+	restorePragmas, err := modifySqLitePragmas(a.db)
+	if err != nil {
+		return 0, err
+	}
+
+	defer func() {
+		if err = restorePragmas(); err != nil {
+			log.Error().Msg(err.Error())
```

Comment:
Using named return values you could make the `importHeaders` return this error. Maybe it's not necessary - if so, just resolve this comment.